src/DGmethods/DGBalanceLawDiscretizations_kernels.jl

```diff
@@ -1,6 +1,20 @@
+using Requires
+@init @require CUDAnative = "be33ccc6-a3ff-5ff2-a52e-74243cff1e17" begin
+  using .CUDAnative
```

Comment:
Is this still required?

Reply:
Yes it is because otherwise `threadIdx` is undefined.

Reply:
Ah! Makes sense